pkg/engine/cosmosdb.go

```diff
@@ -6,7 +6,7 @@ package engine
 func createCosmosDBAccount() map[string]interface{} {
 	cosmosEtcdMap := map[string]interface{}{
 		"apiVersion": "[variables('apiVersionCosmos')]",
-		"name":       "Microsoft.DocumentDB/databaseAccounts",
```

Comment:
The "name" value assignment regressed. It should obviously be assigned the value of the `cosmosAccountName` var we prepare during template generation.